src/core/inc/config_defs.h

```diff
@@ -11,6 +11,7 @@
 #define CONFIG_VCPU_NUM      1
 #define CONFIG_VM_NUM        1
 #define CONFIG_HYP_BASE_ADDR 0
+#define CONFIG_REMIO_DEV_NUM 0
```

Comment:
If this value is set 1, there's no need for `CONFIG_REMIO_DEV_NUM ? CONFIG_REMIO_DEV_NUM : 1` in remio.c and vm.c

Reply:
> If this value is set 1, there's no need for `CONFIG_REMIO_DEV_NUM ? CONFIG_REMIO_DEV_NUM : 1` in remio.c and vm.c

The issue here arises when no remote I/O device is selected by the user, causing the OBJPOOL_ALLOC to be sized at 0. This leads to an error due to ISO C's prohibition on zero-size arrays.

Reply:
I was actually suggesting setting 1 in the script, because it bothers this logic is performed in two places.
But it's not too bad, and the REMIO_DEV_NUM would lose some of its meaning
It's okay as is